src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+stripe.key.publishable=pk_test_VudfQxUpbSS5AmON0qHlCWw3
```

Comment:
All of your keys are in plain text including twilio which is your actual key.  These should be environmental variables.  At this point you should go into twilio and regenerate your keys